dataset/src/main/scala/frameless/functions/NonAggregateFunctions.scala

```diff
@@ -255,6 +266,40 @@ trait NonAggregateFunctions {
   def bin[T](column: AbstractTypedColumn[T, Long]): column.ThisType[T, String] =
     column.typed(sparkFunctions.bin(column.untyped))
 
+  /**
+    * Calculates the MD5 digest of a binary column and returns the value
+    * as a 32 character hex string.
+    *
+    * apache/spark
+    */
+  def md5[T, A](column: AbstractTypedColumn[T, A])(implicit  i1: TypedEncoder[A]): column.ThisType[T, String] =
+    column.typed(sparkFunctions.md5(column.untyped))
+
+  /**
+    * Computes the factorial of the given value.
+    *
+    * apache/spark
+    */
+  def factorial[T](column: AbstractTypedColumn[T, Long])(implicit  i1: TypedEncoder[Long]): column.ThisType[T, Long] =
+    column.typed(sparkFunctions.factorial(column.untyped))
+
+
+  /**
+    * Return true iff the column is NaN.
+    *
+    * apache/spark
+    */
+  def isnan[T, A](column: AbstractTypedColumn[T, A])(implicit  i1: TypedEncoder[A]): column.ThisType[T, Boolean] =
+    column.typed(sparkFunctions.isnan(column.untyped))
+
+  /**
+    * Return true iff the column is Null.
+    *
+    * apache/spark
+    */
+  def isnull[T, A](column: AbstractTypedColumn[T, A])(implicit  i1: TypedEncoder[A]): column.ThisType[T, Boolean] =
```

Comment:
Since null are encoded as `Option[_]`, the equivalent of `isnull` is actually `isNone` and that's also defined in `TypedColumn`.

Reply:
ok I'll remove it but please update to:do list than, since it may confuse other people in future like it confused me.